pom.xml

```diff
@@ -31,50 +27,158 @@
     <url>git@github.com:zanata/openprops.git</url>
   </scm>
 
-	<developers>
-		<!-- in alphabetical order (by id field) -->
-		<developer>
-			<id>sflaniga@redhat.com</id>
-			<name>Sean Flanigan</name>
-			<email>sflaniga@redhat.com</email>
-			<organization>Red Hat</organization>
-			<organizationUrl>http://www.redhat.com/</organizationUrl>
-			<timezone>10</timezone>
-		</developer>
-	</developers>
+  <developers>
+    <!-- in alphabetical order (by id field) -->
+    <developer>
+      <id>sflaniga@redhat.com</id>
+      <name>Sean Flanigan</name>
+      <email>sflaniga@redhat.com</email>
+      <organization>Red Hat</organization>
+      <organizationUrl>http://www.redhat.com/</organizationUrl>
+      <timezone>10</timezone>
+    </developer>
+  </developers>
+  <contributors>
+    <!-- in alphabetical order (by email field) -->
+    <contributor>
+      <name>Ding-Yi Chen</name>
+      <email>dchen@redhat.com</email>
+      <organization>Red Hat</organization>
+      <organizationUrl>http://www.redhat.com/</organizationUrl>
+      <roles>
+	<role>packager</role>
+	<role>tester</role>
+      </roles>
+      <timezone>10</timezone>
+    </contributor>
+  </contributors>
 
+  <properties>
+    <!--
+    Version of java compiler used for building. (Specified as major.minor,
+    as used by javac -source and -target).
+    -->
+    <required.java>1.8</required.java>
+    <!--
+      Version of Java used for runtime (major.minor) and for bytecode
+      generation. Should correspond with animal.sniffer.signature.
+    -->
+    <required.jvm>1.7</required.jvm>
+  </properties>
 
   <build>
-      <plugins>
-          <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-gpg-plugin</artifactId>
-              <configuration>
-                  <useAgent>true</useAgent>
-              </configuration>
-          </plugin>
-          <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-compiler-plugin</artifactId>
-              <version>2.0.2</version>
-              <configuration>
-                  <source>1.5</source>
-                  <target>1.5</target>
-                  <showDeprecation>true</showDeprecation>
-                  <showWarnings>true</showWarnings>
-                  <optimize>true</optimize>
-              </configuration>
-          </plugin>
-      </plugins>
+    <plugins>
+      <plugin>
+	<groupId>org.apache.maven.plugins</groupId>
```

Comment:
We should probably re-indent this section. You seem to be using two spaces everywhere else, which should be fine.